src/napari_builtins/_tests/test_io.py

```diff
@@ -339,3 +340,24 @@ def test_add_many_zarr_1d_array_is_ignored(tmp_path):
         else:
             assert isinstance(out[0], da.Array), name
             assert out[0].ndim == int(name[0])
+
+
+def test_github_and_gitlab_to_raw_url():
```

Comment:
Parameterizing the test cases would make it easier to extend tests for different URL structures.

Reply:
done